# Complete scene graph with objects and relations

std_msgs/Header header
DetectedObject[] objects
SpatialRelation[] relations
uint32 total_objects
string room_context
